nextjs-legacy/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "scripts": {
-    "dev": "NODE_OPTIONS='-r dotenv/config --loader newrelic/esm-loader.mjs -r newrelic' next dev",
+    "dev": "NODE_OPTIONS='-r dotenv/config --loader newrelic/esm-loader.mjs -r newrelic -r @newrelic/next' next dev",
```

Comment:
you don't need to load both agent and `@newrelic/next`. `@newrelic/next` loads agent